lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -187,15 +187,16 @@ class ReportDownloads
       forceFitColumns: true
 
     columns = [
-      id: 'link'
-      field: 'link'
-      name: gettext('File Name')
-      toolTip: gettext("Links are generated on demand and expire within 5 minutes due to the sensitive nature of student information.")
-      sortable: false
-      minWidth: 150
-      cssClass: "file-download-link"
-      formatter: (row, cell, value, columnDef, dataContext) ->
-        '<a href="' + dataContext['url'] + '">' + dataContext['name'] + '</a>'
+        id: 'link'
+        field: 'link'
+        name: gettext('File Name')
+        toolTip: gettext("Links are generated on demand and expire within 5 minutes
+                          due to the sensitive nature of student information.")
+        sortable: false
+        minWidth: 150
+        cssClass: "file-download-link"
+        formatter: (row, cell, value, columnDef, dataContext) ->
+          '<a class="course-forums-data" href="' + dataContext['url'] + '">' + dataContext['name'] + '</a> (<span class="delete-report">Delete Report</span>)'
```

Comment:
maybe add an x-icon ? similar to https://github.com/Stanford-Online/edx-platform/blob/master/lms/static/coffee/src/instructor_dashboard/membership.coffee#L123